src/compas/_json.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import json
+from compas.utilities import DataEncoder
+from compas.utilities import DataDecoder
+
+
+__all__ = [
+    'json_dump',
+    'json_dumps',
+    'json_load',
+    'json_loads'
+]
+
+
+def json_dump(data, fp):
+    """Write a collection of COMPAS object data to a JSON file.
+
+    Parameters
+    ----------
+    data : list of :class:`compas.base.Base`
```

Comment:
Are really only `list`s supported? isn't it also valid to pass a single instance of a class implementing anything that can be encoded/decoded by the `DataEncoder/DataDecoder` pair and it will work?

Reply:
yes indeed. will update...

Reply:
would be nice to support dictionaries as well something like {obj_a: ..., obj_b: ...}

Reply:
just to be clear. you can in fact pass it anything you want. it is just the documentation that suggests the input is limited to lists...